Cargo.toml

```diff
@@ -223,3 +224,6 @@ unexpected_cfgs = { level = "warn", check-cfg = [
   'cfg(zcash_unstable, values("zfuture", "nu7"))',
   'cfg(live_network_tests)',
 ] }
+
+[patch.crates-io]
+zcash_script = { git = "https://github.com/ZcashFoundation/zcash_script.git", rev = "96363951c0fade4f99446d2a847efe706deff35c" }
```

Comment:
Current tip of https://github.com/ZcashFoundation/zcash_script/pull/247; this will change before this PR merges.